src/Components/test/testassets/ComponentsApp.Server/ComponentsApp.Server.csproj

```diff
@@ -11,6 +11,7 @@
   <ItemGroup>
     <Reference Include="Microsoft.AspNetCore.Components.Server" />
     <Reference Include="Microsoft.AspNetCore.Mvc" />
+    <Reference Include="Microsoft.AspNetCore.StaticFiles" />
```

Comment:
?

Reply:
I don't think this is needed anymore. I had to add static files back to `Components.Server`.